### ISSUE TITLE
Conflicts in query/fields names are not handled

### ISSUE THREAD

**Issue by moufmouf** [thecodingmachine/graphqlite#175]
This code is currently valid:

```php
    /**
     * @Query(name="getProduct")
     */
    public function getProduct(): SpecialProduct
    {
        return new SpecialProduct('Special box', 10.99);
    }

    /**
     * @Query(name="getProduct")
     */
    public function getProducts2(): ArrayIterator
    {
        return new ArrayIterator([new SpecialProduct('Special box', 10.99), new SpecialProduct('Special box', 10.99)]);
    }
```

Note: how the "name" attribute is used to pass the same query name twice.
This should throw an exception.

**Comment by flow-control** [comment]
Hey David,

would you accept a pull request for this? I think i could handle this in the next two weeks, any additional remarks?

/Flo

**Comment by moufmouf** [comment]
Oh definitely, I did not start working on this. If you want to give it a try; with pleasure!

**Comment by moufmouf** [comment]
Closed by #181 and #194 
Thanks @flow-control !
